files/readme.md

```diff
@@ -0,0 +1,8 @@
+# about your new image collection
+
+The `imagecollection.csv` file contains metadata for two objects:
+
+- an Image object
+- a Collection object that contains the Image
+
+It has no errors, and will validate against the [metadata-ci script](https://github.com/ucsblibrary/metadata-ci)!
```

Comment:
First exercise: update this to say IM SORRY WE FOUND AN ERROR WE"RE WORKING ON IT

Reply:
second exercise is fix it